llvmlite/ir/instructions.py

```diff
@@ -162,7 +162,7 @@ def descr(self, buf):
 class PredictableInstr(Instruction):
 
     def set_weights(self, weights):
-        operands = [MetaDataString(self.module, "branch_weights")]
+        operands = [MetaDataString(self.module, b"branch_weights")]
```

Comment:
No, this is completely wrong. What it "feels" is irrelevant. What matters is exactly one thing: how we consider `.ll` files to be encoded. There are two (reasonable) options:
 1. `.ll` files are ASCII, and UTF-8 cannot be put into them.
 2. `.ll` files are UTF-8-encoded.
`master` has a random mixture of these two. This patchset currently fixes it to represent option 1. If that's undesirable then it should be fixed to use option 2 (which I prefer anyway, since the instruction names are human-readable and LLVM doesn't care what's in there, and putting a `unicode` into a character array has a perfectly valid use, namely emitting UTF-8 string literals in a frontend).